packages/create-webpack-app/bin/cli.js

```diff
@@ -0,0 +1,55 @@
+import { Command } from "commander";
+import { resolve, join } from "path";
+import nodePlop from "node-plop";
+// Cspell:ignore plopfile, plopfile.js
+
+const program = new Command();
+const plop = await nodePlop("./lib/plopfile.js");
+const defaultValues = {
+  init: {
+    projectPath: resolve(join(process.cwd(), ".")),
```

Comment:
Writing this could be cumbersome, how about default to scaffold in the current dir? I think it's a standard to scaffold in the current dir.

Thoughts? @snitin315 @evenstensberg

Reply:
Yeah, we don't have to ask the project path we can just scaffold files in `cwd/project-name`.

Reply:
in a recent local commit I have changed the default projectPath to process.cwd() so the generator defaults to 
cwd/project-name, 
but do you want me to remove the prompt where it asks generation path?(because that is important for bypassing with argument for genPath)

Reply:
I think if the user wants to define a custom path they can just name the project like - `folder1/folder2/project-name` This will create this file hierarchy in the current cwd

Reply:
What should I name this ?
genPath? or name?